dali/pipeline/operators/op_schema.h

```diff
@@ -255,6 +264,10 @@ class DLL_PUBLIC OpSchema {
     return allow_multiple_input_sets_;
   }
 
+  DLL_PUBLIC inline bool AllowsSequences() const {
```

Comment:
I would expose this function in the `backend_impl.cc` so ops.py can take that into accout generating docs for given operator

Reply:
will do

Reply:
done